docs/usage/create.rst

```diff
@@ -74,3 +74,14 @@ Examples
     $ cd /home/user/Documents
     # The root directory of the archive will be "projectA"
     $ borg create /path/to/repo::daily-projectA-{now:%Y-%m-%d} projectA
+
+    # Use external command to determine files to archive
+    # Use --paths-from-stdin with find to only backup files less than 1MB in size
+    $ find ~ -size -1000k | borg create --paths-from-stdin /path/to/repo::small-files-only
+    # Use --paths-from-command with find to only backup files from a given user
+    $ borg create --paths-from-command -- /path/to/repo::joes-files find ~ -user joe
```

Comment:
is that correct? would it also work like:
```
borg create --paths-from-command='find ~ -user joe' /path/to/repo::joes-files
```

Reply:
yeah that's correct I tested it

Reply:
note: the command i suggested does **not** work, strange. i'll have to check whether this is consistent with similar commands.

but i have checked this, it works and looks a bit more easy maybe?
```
borg create --paths-from-command /path/to/repo::joes-files -- find ~ -user joe
```

note: the special argument "--" forces the end of option-scanning (borg shall option-scan the borg command, but not the find command).

another issue with that command is that it makes little sense usually: as the starting point for find is the home directory of the current user (who might be joe or not joe), usually all or no files will be owned by joe.

so, if we want to demonstrate `-user`, I suggest to use /srv/samba/shared as the starting point (which is a more usual place to be shared by different users). or we use something other than `-user`.